### ISSUE TITLE
getting error while using sentry.init function

### ISSUE THREAD

**Issue by kaurrJaspreet** [getsentry/sentry-react-native#1203]
**OS:**
- [ ]  MacOS

**Platform:**
- [x]  iOS
- [ ]  Android

**SDK:**
- [x]  `@sentry/react-native` (>= 1.0.0)

**SDK version:** 0.0.0

**`react-native` version:** 0.63

**Are you using Expo?**
- [x]  Yes

I am getting error below:
TypeError: global.document.addEventListener is not a function. (In 'global.document.addEventListener('click', domEventHandler('click', triggerHandlers.bind(null, 'dom')), false)', 'global.document.addEventListener' is undefined)

inside "node_modules/@sentry/utils/dist/instructions.js" file on line number 328 and 329.

Please help me in this.
I just installed "npm install --save @sentry/react-native";
cd ios
pod install

**Comment by jennmueng** [comment]
This is highly unusual to say the least, that function should not be called if `global.document` doesn't exist, and it shouldn't exist at all on react native. Are you setting it somewhere?

**Comment by jennmueng** [comment]
Looks like there haven't been any updates for a while, closing this for now. Let me know if the issue persists.

**Comment by DangerBlack** [comment]
I've experience the exact same behavior today in my production build.
In order to fix it I had to manually write this before running `init`

`global.document.addEventListener = global.document.addEventListener ? global.document.addEventListener : () => {};`

**Comment by nickL** [comment]
Experiencing the same on latest build (4.2.0). Can we re-open this issue?

**Comment by nickL** [comment]
I was able to fix this for 4.2.2 by disabling the `dom` option in the default  Breadcrumbs integrations passed to init:

```js
   const config = {
    ...configOptions,
    integrations: [new Sentry.BrowserIntegrations.Breadcrumbs({ dom: false })],
  };
```
